Makefile

```diff
@@ -329,6 +388,7 @@ DOCKER_GO_BUILD := mkdir -p .go-pkg-cache && \
                               -e LOCAL_USER_ID=$(MY_UID) \
                               -v $${PWD}:/go/src/github.com/projectcalico/felix:rw \
                               -v $${PWD}/.go-pkg-cache:/go/pkg:rw \
+                              -v $${HOME}/.docker:/home/user/.docker \
```

Comment:
What's this change for?

Reply:
Ah, this is a relic from when this PR included multi-arch manifest. We decided to punt on it for now, doing everything short of it, so this should be removed. Will fix.